test/e2e/cloud/nodes.go

```diff
@@ -47,8 +47,9 @@ var _ = SIGDescribe("[Feature:CloudProvider][Disruptive] Nodes", func() {
 		nodeDeleteCandidates := framework.GetReadySchedulableNodesOrDie(c)
 		nodeToDelete := nodeDeleteCandidates.Items[0]
 
-		origNodes, err := e2enode.GetReadyNodesIncludingTaintedOrDie(c)
+		origNodes, err := e2enode.GetReadyNodesIncludingTainted(c)
 		framework.ExpectNoError(err)
```

Comment:
Here I didn't replace with `gomega.Expect(err).NotTo(gomega.HaveOccurred())`, since it is called directly in a ginkgo test suit.  Please comment if I should replace `ExpectNoError` with `gomega.Expect(err)`

Reply:
What do you mean by "called directly in a ginkgo test suit"?

My thinking is that for all the methods you rewrote to return an error (instead of calling `framework.ExpectNoError()` within them) should call `gomega.Expect(err).NotTo(gomega.HaveOcurred())`.
Otherwise the `framework.ExpectNoError(err)` will apply an "offset" and will return (upon an error) the line of the calling method (i.e., func calling `e2enode.GetReadyNodesIncludingTainted`) instead of this method (`e2enode.GetReadyNodesIncludingTainted`).

Reply:
Got it